bin/setup

```diff
@@ -2,20 +2,24 @@
 
 # Run this script immediately after cloning the codebase.
 
+# Squash all commits to get a clean slate
+git update-ref -d refs/heads/master
+git commit -am "New Proteus-Middleman project"
+
 # Make sure Bundler is installed
 if [ `gem query -i -n bundler` = "false" ]; then
```

Comment:
The docs made it sound like it wasn't a default, but it does seem to work exactly the same without it. :+1: Fixed